src/App.tsx

```diff
@@ -15,36 +16,93 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  ALPHABET = 'alphabet',
+  LENGTH = 'length',
+  DEFAULT = '',
+}
+
+function sortGoods(
+  goods: string[],
+  sortField: SortType,
+  reversed: boolean,
+): string[] {
+  const sortedGoods = [...goods].sort((a, b) => {
+    if (sortField === SortType.ALPHABET) {
+      return a.localeCompare(b);
+    }
+
+    if (sortField === SortType.LENGTH) {
+      return a.length - b.length;
+    }
+
+    return 0;
+  });
+
+  return reversed ? sortedGoods.reverse() : sortedGoods;
+}
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType>(SortType.DEFAULT);
+  const [reversed, setReversed] = useState(false);
```

Comment:
```suggestion
  const [isReversed, setIsReversed] = useState(false);
```

If the value contains `boolean` value, then the name of variable should start with `is`